src/components/payment-options.js

```diff
@@ -67,18 +67,26 @@ var StripeButton = React.createClass({
     name: React.PropTypes.string.isRequired,
     currency: React.PropTypes.object.isRequired
   },
-  onChange: function() {
+  onClick: function() {
     if (this.props.onClick) {
       this.props.onClick();
     }
-    setTimeout(() => {
-      this.props.onSubmit(this.props.validate, this.props.submit);
-    });
+    this.props.onSubmit(this.props.validate, this.props.submit);
     reactGA.event({
       category: "User Flow",
       action: "Stripe Clicked"
     });
   },
+  componentDidMount: function() {
+    // Need this because Chrome on iOS plus StripeCheckout triggers a popup.
+    // We need to ensure the popup isn't blocked.
+    this.input.addEventListener("click", this.onClick);
```

Comment:
Need to do this directly on the DOM. I think React is doing some callbacks internally or something?